erpnext/buying/doctype/purchase_order/purchase_order.py

```diff
@@ -443,6 +443,9 @@ def update_item(obj, target, source_parent):
 			"doctype": "Purchase Taxes and Charges",
 			"add_if_empty": True
 		},
+		"Payment Schedule": {
```

Comment:
@marination  Payment Terms are intentionally not copied from Purchase Order to Invoice. The reason behind this is Payment Terms vary from business to business. Some people take payment at the PO level and other at the Invoice level and since Payment Term's due date is are based on invoice date, while coping payment terms same date is copied but PO and PI might have different  posting date which makes the Payment terms Incorrect. So to summarise its intentional